price-server/src/exchange_tick_cache.rs

```diff
@@ -14,6 +14,7 @@ pub struct ExchangeTickCache {
     config: ExchangePriceCacheConfig,
 }
 
+#[allow(dead_code)]
```

Comment:
added this because the methods are used in tests

Reply:
Then why not `[cfg(test)]`?